doc/modules/mod_muc_light_commands.md

```diff
@@ -0,0 +1,49 @@
+# MongooseIM's multi-user chat light commands set
+
+## Purpose
+This is a set of commands, providing actions related to multi-user chat light features.
+It is currently used to expose the part of REST API responsible for managing rooms.
+
+## Configuration
+This module contains command definitions which are loaded when the module is activated.
+There are no options to be provided, therefore the following entry in the config file is sufficient:
+
+```
+{mod_muc_light_commands, []}
+```
+
+## Commands
+This file consists of [commands definitions](mod_commands/).
+Following commands (along with functions necessary for them to run) are defined:
++ `create_muc_light_room`  
+Creates a MUC Light room.  
+    Args:  
+        - `domain` (binary)  
+        - `name`  (binary)  
+        - `owner` (binary)  
+        - `subject` (binary)  
++ `create_identifiable_muc_light_room`  
+Updates a MUC Light room.  
```

Comment:
Nope. Creates a MUC Light room with user-provided username part in JID (while the "create" above picks unique username).

The description in the file is invalid, please fix it as well. :)